torch/testing/_internal/common_methods_invocations.py

```diff
@@ -516,6 +516,7 @@ def __init__(self,
                  # the following metadata relates to sparse support and is used in test_sparse.py
                  supports_sparse=False,  # whether the op supports sparse inputs
 
+                 skip_jit_scripting=False,  # only run tracing tests
```

Comment:
Should we model this directly as "supports_scripting"?

Typically skip metadata should be in a SkipInfo. That would work if the above test was split into tracing and scripting versions, which might make more sense than modeling a partial test skip as operator metdata.

Reply:
yea, i agree it would be nice to eventually split these tests up, but I would prefer to prioritize expanding op coverage first. Fine with `supports_scripting`. There are a lot of fixes I think are higher-pri than splitting them up, I have a WIP PR to improve autodiff for instance and there are a ton of skipped JIT tests for pretty small reasons.

Reply:
That sounds good

Reply:
let me actually try to rope more people in to fixing this stuff, and maybe we can split up tracing and scripting sooner..